### PR TITLE
add ocrd_wrap

### PR DESCRIPTION
**Pull request by bertsky**
This gives:
- ocrd-preprocess-image
- ocrd-skimage-normalize
- ocrd-skimage-denoise-raw
- ocrd-skimage-binarize
- ocrd-skimage-denoise

### CODE REVIEW

**Comment by kba** [comment]
:+1: very useful and versatile project. but please wait with merge until i have set up the release mechanism discussed in #101 (hopefully later today).

File: .gitmodules [hunk 1]
```diff
@@ -76,3 +76,6 @@
 [submodule "ocrd_pagetopdf"]
 	path = ocrd_pagetopdf
 	url = https://github.com/UB-Mannheim/ocrd_pagetopdf.git
+[submodule "ocrd_wrap"]
+	path = ocrd_wrap
+	url = https://github.com/bertsky/ocrd_wrap
```
Comment (stweil) [inline .gitmodules:81]:
```suggestion
	url = https://github.com/bertsky/ocrd_wrap.git
```

**Comment by bertsky** [comment]
@kba could you do the merging/releasing for now?

**Comment by kba** [comment]
On it. But I want to test that the release script is working properly and unfortunately "adding a new submodule" is not covered by the changelog. Merging with the instructions in the contributing guideline also need to include `git submodule update --init`. I will merge this today but first, I want to make sure that a "update-submodules" PR is mergeable.

**Comment by kba** [comment]
Released as https://github.com/OCR-D/ocrd_all/releases/tag/v2020-06-12

🤞 that the tag aliassing works in Circle CI.

**Pull request merged**